hana/install_pydbapi.sls

```diff
@@ -0,0 +1,46 @@
+{%- from "hana/map.jinja" import hana with context -%}
+{% set host = grains['host'] %}
+
+{% for node in hana.nodes if node.host == host %}
+{% if loop.first %}
+{% set pydbapi_output_dir = '/tmp/pydbapi' %}
+hana_install_python_pip:
+  pkg.installed:
+    {% if grains['pythonversion'][0] == 2 %}
+    - name: python-pip
+    {% else %}
+    - name: python3-pip
+    {% endif %}
+    - retry:
+        attempts: 3
+        interval: 15
+    - resolve_capabilities: true
+
+hana_extract_pydbapi_client:
```

Comment:
@arbulu89 it has the same code as the NW formula. Would it make more sense to move to a generic state on salt-shaptools and use it on both?

Reply:
What do you exactly mean? The whole sls file?
I thought about that, but I prefer to have the logic visible. I don't want to install and run things like `pip` hidden in some module.
It doesn't require much more maintenance, and by now, the code duplication is minimal.